homeassistant/components/lutron/light.py

```diff
@@ -25,11 +32,41 @@ async def async_setup_entry(
     Adds dimmers from the Main Repeater associated with the config_entry as
     light entities.
     """
+    ent_reg = er.async_get(hass)
     entry_data: LutronData = hass.data[DOMAIN][config_entry.entry_id]
+    lights = []
+
+    for area_name, device in entry_data.lights:
+        if device.type == "CEILING_FAN_TYPE":
+            # If this is a fan check to see if this entity already exists. If not, do not create a new one.
+            entity_id = ent_reg.async_get_entity_id(
+                Platform.LIGHT,
+                DOMAIN,
+                slugify(f"{entry_data.client.guid} {device.uuid}"),
```

Comment:
```suggestion
                f"{entry_data.client.guid}_{device.uuid}",
```
No need to slugify, as we also don't do that for the unique id